CONTRIBUTING.adoc

```diff
@@ -93,12 +93,12 @@ If you would like these changes to be stored on your personal remote repository,
 
 `git push me mywork`
 
-Once you are satisified with your work, there is one last step to complete before submitting the pull request - clean up the history.
+Once you are satisfied with your work, there is one last step to complete before submitting the pull request - clean up the history.
 
 [[gh-history-cleanup]]
 === Clean up your git commit history
 
-Commit history can often be full of temporiariy commit messages, of commits with code changes that ultimately didn't make the final cut.
+Commit history can often be full of temporarily commit messages, of commits with code changes that ultimately didn't make the final cut.

```

Comment:
You mean temporary.  Also prefer
    , of commits
=>
  , and of commits

Reply:
and/or - best of no worlds. I'll make a quick change.

Reply:
Sure enough, I could edit and commit directly to your PR branch using the github interface. Slick.